contracts/src/v0.8/automation/dev/v2_3/AutomationRegistrar2_3.sol

```diff
@@ -107,7 +107,7 @@ contract AutomationRegistrar2_3 is TypeAndVersionInterface, ConfirmedOwner, IERC
 
   LinkTokenInterface public immutable i_LINK;
   IWrappedNative public immutable i_WRAPPED_NATIVE_TOKEN;
-  IAutomationRegistryMaster2_3 s_registry;
+  IAutomationRegistryMaster2_3 private s_registry;
```

Comment:
Default visibility is internal, should this be internal instead?

Reply:
They use private for all others below, and it isn't used outside of this contract